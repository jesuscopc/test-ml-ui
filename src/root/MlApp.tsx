import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import Routes from './routes';

const MlApp = (): React.ReactElement => {
  return (
    <BrowserRouter basename="/">
      <Routes />
    </BrowserRouter>
  );
};

export default MlApp;
