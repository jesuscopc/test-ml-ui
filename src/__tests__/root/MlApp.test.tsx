import React from 'react';
import { shallow } from 'enzyme';
import MlApp from 'root/MlApp';

describe('Suit test MlApp', () => {
  let wrapper = shallow(<MlApp />);
  beforeEach(() => {
    wrapper = shallow(<MlApp />);
  });
  test('should be defined MLApp', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
