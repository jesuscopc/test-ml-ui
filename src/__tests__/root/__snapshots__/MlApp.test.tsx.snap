// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Suit test MlApp should be defined MLApp 1`] = `
<BrowserRouter
  basename="/"
>
  <Routes />
</BrowserRouter>
`;
